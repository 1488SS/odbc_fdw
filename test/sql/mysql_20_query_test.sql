SELECT * FROM mysql_test_table;
SELECT * FROM query_mysql_test_table;
SELECT * FROM ODBCTablesList('mysql_fdw', 1);