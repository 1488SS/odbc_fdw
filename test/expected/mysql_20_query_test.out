SELECT * FROM mysql_test_table;
 id | varchar_example | text_example | integer_example | numeric_example |    timestamp_example     | boolean_example 
----+-----------------+--------------+-----------------+-----------------+--------------------------+-----------------
  1 | example         | example      |             100 |              10 | Fri Jan 01 00:00:00 2016 |               1
(1 row)

SELECT * FROM query_mysql_test_table;
 count(1) 
----------
        1
(1 row)

SELECT * FROM ODBCTablesList('mysql_fdw', 1);
 schema |       name       
--------+------------------
        | mysql_test_table
(1 row)

SELECT * FROM ODBCTableSize('mysql_fdw', 'mysql_test_table');
 odbctablesize 
---------------
             1
(1 row)

SELECT * FROM ODBCQuerySize('mysql_fdw', 'select * from mysql_test_table');
 odbcquerysize 
---------------
             1
(1 row)

